### PR TITLE
Add inter-op BBE's for array, vararg, fields

### PR DESCRIPTION
**Pull request by manuranga**

### CODE REVIEW

File: examples/access-mutate-java-fields/access-mutate-java-fields.description [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Java fields (both static and instance) can be read and modified from Ballerina.
```
Comment (MaryamZi) [inline examples/access-mutate-java-fields/access-mutate-java-fields.description]: Each line in the description files has to start with `// `
```suggestion
// Java fields (both static and instance) can be read and modified from Ballerina.
```

Comment (MaryamZi) [inline examples/access-mutate-java-fields/access-mutate-java-fields.description]: The .description file name should be the same as the folder name, with only the `-`s replaced with `_`s.

e.g., access_mutate_java_fields.description

Shall we also update the .bal and .out similarly?

File: examples/access-mutate-java-fields/access-mutate-java-fields.out [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# To start the service, navigate to the directory that contains the
```
Comment (MaryamZi) [inline examples/access-mutate-java-fields/access-mutate-java-fields.out]:
```suggestion
# To run this sample, navigate to the directory that contains the
```

File: examples/java-arrays/java-arrays.out [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# To start the service, navigate to the directory that contains the
```
Comment (MaryamZi) [inline examples/java-arrays/java-arrays.out]:
```suggestion
# To run this sample, navigate to the directory that contains the
```

File: examples/java-varargs/java-varargs.out [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# To start the service, navigate to the directory that contains the
```
Comment (MaryamZi) [inline examples/java-varargs/java-varargs.out]:
```suggestion
# To run this sample, navigate to the directory that contains the
```

**Comment by manuranga** [comment]
Thanks @MaryamZi did the fixes, please re-revew.

File: examples/java-varargs/java_varargs.bal [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import ballerina/io;
+import ballerinax/java;
+
+public function main() {
+    var answer = 42;
+    // External vararg function can be called like any other Ballerina vararg function
```
Comment (MaryamZi) [inline examples/java-varargs/java_varargs.bal]:
```suggestion
    // An external vararg function can be called like any other Ballerina vararg function.
```

File: examples/java-varargs/java_varargs.bal [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+    // If a method is non-static, receiver instance has to be provided as the first parameter
```
Comment (MaryamZi) [inline examples/java-varargs/java_varargs.bal]:
```suggestion
    // If a method is non-static, the receiver instance has to be provided as the first parameter.
```

File: examples/java-varargs/java_varargs.bal [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+    var javaList = asList(1, 2, answer);
+    io:println(javaList);
+}
+
+// Defines a Ballerina function whose implement is provided by an external Java method.
```
Comment (MaryamZi) [inline examples/java-varargs/java_varargs.bal]:
```suggestion
// Define a Ballerina function whose implementation is provided by an external Java method.
```

File: examples/java-varargs/java_varargs.bal [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+// When vararg is used in the Ballerina function signature, corresponding values are wrapped with
```
Comment (MaryamZi) [inline examples/java-varargs/java_varargs.bal]:
```suggestion
// When a vararg is used in the Ballerina function signature, corresponding values are wrapped in
```

File: examples/java-varargs/java_varargs.description [hunk 9]
```diff
@@ -0,0 +1,3 @@
+// Java varargs functions (both static and instance) can be invoked from Ballerina.
+// If there are overloaded Java method and you want to remove the ambiguity, `paramTypes` field can be used
```
Comment (MaryamZi) [inline examples/java-varargs/java_varargs.description]:
```suggestion
// If there are overloaded Java methods and you want to remove the ambiguity, the `paramTypes` field can be used
```

**Comment by manuranga** [comment]
@MaryamZi Thank you very much for the fixes.

**Pull request merged**